File: inheritance.py
class Shape:
    def setcolour(self, colour):
        self.colour = colour

class Circle(Shape):
    def setradius(self, radius):
        self.radius = radius
    def area(self):
        return 3.14 * self.radius * self.radius
    
c = Circle()
c.setcolour("Red")
c.setradius(5.0)
print(c.area())

File: inheritance.java
class Shape{
    String colour;
    void setcolour(String c){ colour=c}
}
class Circle extends Shape{
    double radius;
    void setradius(double r){ radius=r; }
    double area(){ return 3.14*radius*radius; }
}

public class Main {
    public static void main(String[] args) {
        Circle c = new Circle();
        c.setcolour("Red");
        c.setradius(5.0);
        System.out.println("Colour: " + c.colour);
        System.out.println("Area: " + c.area());
    }
}